docs/sql/statements/create-user.rst

```diff
@@ -79,3 +79,26 @@ The following ``user_parameter`` are supported to define a new user account:
   ::
 
      CREATE USER john password 'foo'
+
+.. vale off
+
+.. _create-user-jwt:
+
+:jwt:
+  JWT properties map ('iss' and 'username') entered as string literal. e.g.::
+
+     CREATE USER john WITH (jwt = {"iss" = 'https://login.company.org/keys', "username" = 'test@company.org'})
```

Comment:
Please use example.com as domain name, see https://www.rfc-editor.org/rfc/rfc6761.html

Reply:
updated and squashed into first commit